admin/dw-options.php

```diff
@@ -58,6 +58,22 @@ function dw_render_options_page() {
 
 			<h2 class="title">Standaard waarden voor nieuwe cursussen</h2>
 			<table class="form-table">
+				<tr>
+					<th scope="row">
+						<label class="description" for="dw_settings[ccvCode]">
```

Comment:
The other `dw_settings` values seem to have underscores. Change this to `ccv_code`?